JS.md

```diff
@@ -166,6 +166,29 @@ Otherwise, this function performs synchronous compilation of the `BufferSource`:
   the validated `Ast.module`.
 * On failure, a new `WebAssembly.CompileError` is thrown.
 
+### `WebAssembly.Module.exports`
```

Comment:
Just evaluating it as a JS API, though, I think the plural makes more sense and matches other APIs that return an aggregate like `Object.keys`, `Response.headers`, `Node.childNodes`.  How we name the section headers is just a spec-internal detail anyway and could change at any time.  IIRC, some of the section names even started as plural and the s's were stripped when the string names were embedded in the section header to save bytes.